benchmark/utils/CxxTests/CxxStdlibPerformance.h

```diff
@@ -4,11 +4,26 @@
 #include <vector>
 #include <set>
 
+// FIXME swift-ci linux tests do not support std::span
+#if defined(__has_include) && __has_include(<span>)
```

Comment:
Minor: could you please deduplicate this check by doing something like this?
```
#if defined(__has_include) && __has_include(<span>)
#define SPAN_DEFINED 1
#else
#define SPAN_DEFINED 0
#endif
```
and then using `#if SPAN_DEFINED` to do the check?

Reply:
Sure!